test/unit/uk/gov/hmrc/exports/base/MockMetrics.scala

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package unit.uk.gov.hmrc.exports.base
+
+import com.codahale.metrics.MetricRegistry
+import com.kenshoo.play.metrics.Metrics
+
+trait MockMetrics {
+
+  val metricRegistry = new MetricRegistry()
+
+  val metrics: Metrics = new Metrics {
+    override def defaultRegistry: MetricRegistry = metricRegistry
+
+    override def toJson: String = ???
```

Comment:
Is it intentional?

Reply:
Yes - this is used by framework for reporting and in client code is unused - maybe i should make little scaladoc about this ?